react-shopping-cart/src/redux/reducers/auth.reducer.js

```diff
@@ -0,0 +1,31 @@
+const { LOGIN_USER, LOGOUT_USER } = require('redux/actions/auth.action');
+
+const initialState = {
+  accessToken: '',
+  name: '',
+  email: '',
+  address: '',
+  phoneNumber: {
+    first: '',
+    second: '',
+  },
+};
+
+function auth(state = initialState, action) {
+  switch (action.type) {
+    case LOGIN_USER:
+      const { phone, ...rest } = action.payload;
+      const phoneNumbers = phone.split('-');
+      const phoneNumber = {
+        first: phoneNumbers[1],
+        second: phoneNumbers[2],
+      };
```

Comment:
(마이너) index로 접근해주는 것 보다 배열에 할당하면 010-중간번호-끝번호 형태가 유추된다고 생각되네요!
```suggestion
      const [_, first, second] = phone.split('-');
      const phoneNumber = {
        first,
        second
      };
```